arena.go

```diff
@@ -176,7 +182,14 @@ func receive(bot *Bot) string {
     return line
 }
 
-func pick_regions(state *State, bots []*Bot, regions []int64) {
+func send_map(state *State, bots []*Bot) {
+
+}
+
+func pick_regions(state *State, bots []*Bot) {
+    regions := make([]int64, len(state.starting_regions), cap(state.starting_regions))
```

Comment:
Does a capacity check. Could also use length. I think there might not be a difference for arrays.